variables.tf

```diff
@@ -669,11 +669,8 @@ variable "runner_worker_docker_autoscaler_asg" {
     health_check_type = Controls how health checking is done. Values are - EC2 and ELB
     instance_refresh_min_healthy_percentage = The amount of capacity in the Auto Scaling group that must remain healthy during an instance refresh to allow the operation to continue, as a percentage of the desired capacity of the Auto Scaling group.
     instance_refresh_triggers = Set of additional property names that will trigger an Instance Refresh. A refresh will always be triggered by a change in any of launch_configuration, launch_template, or mixed_instances_policy.
-    max_growth_rate = The maximum number of machines that can be added to the runner in parallel.
     on_demand_base_capacity = Absolute minimum amount of desired capacity that must be fulfilled by on-demand instances.
     on_demand_percentage_above_base_capacity = Percentage split between on-demand and Spot instances above the base on-demand capacity.
-    override_instance_types = List to override the instance type in the Launch Template. Allow to spread spot instances on several types, to reduce interruptions
-    profile_name = profile_name = Name of the IAM profile to attach to the Runner Workers.
```

Comment:
issue: as long as we do not change the type definition below, we mustn't remove the description.

suggestion: change the description to `(deprecated, not in use) # TODO remove in the upcoming major release` or similar.